src/ldp.rs

```diff
@@ -1133,6 +1180,133 @@ impl ProofSuite for Eip712Signature2021 {
     }
 }
 
+#[cfg(feature = "keccak-hash")]
+pub struct EthereumPersonalSignature2021;
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
+#[cfg(feature = "keccak-hash")]
+impl ProofSuite for EthereumPersonalSignature2021 {
+    async fn sign(
```

Comment:
Not a fan of duplicating the crypto code but I suppose it's the best solution currently.